CoffeeChat/Controller/EditTimeAvailabilityViewController.swift

```diff
@@ -159,9 +153,71 @@ class EditTimeAvailabilityViewController: UIViewController {
     @objc private func backPressed() {
         navigationController?.popViewController(animated: true)
     }
+
+    private func stringTimeToFloat(time: String) -> Float {
```

Comment:
Since `stringTimeToFloat` and `floatToStringTime` both deal with Time strings/floats/data, they might be better in `Time+Helper.swft`